sormas-api/src/main/resources/strings.properties

```diff
@@ -697,6 +697,7 @@ headingShowExternalMessage = Message
 headingSelfReportSideComponent = Self reports
 headingSignsAndSymptoms = Clinical Signs and Symptoms
 headingSimilarImmunization = Similar immunizaton
+headingSimilarPerson = Similar persons found based on person introduced data
```

Comment:
This sentence doesn't seem correct, I think we should be precise and say There are some other persons with ht e same national health id.

Reply:
changed